services/queue/src/queueservice.js

```diff
@@ -144,6 +144,10 @@ class QueueService {
     // Keep a cache of pending counts as mapping:
     //    <provisionerId>/<workerType> -> {lastUpdated, count: promise}
     this.countPendingCache = {};
+
+    // Keep a record of last claimed for each queue as mapping:
+    //    <provisionerId>/<workerType> -> lastClaimed
+    this.lastClaimedMap = {};
```

Comment:
We currently run 25 instances of the REST API server, so yeah, this single-process map won't work :)